test/TestProjects/MgmtMockAndSample/tests/Generated/Samples/Sample_DiskEncryptionSetCollection_CreateOrUpdate_CreateADiskEncryptionSet.cs

```diff
@@ -0,0 +1,61 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System;
+using System.Threading.Tasks;
+using Azure;
+using Azure.Core;
+using Azure.Identity;
+using Azure.ResourceManager;
+using Azure.ResourceManager.Models;
+using Azure.ResourceManager.Resources;
+using MgmtMockAndSample.Models;
+
+namespace MgmtMockAndSample
+{
+    public partial class Sample_DiskEncryptionSetCollection_CreateOrUpdate_CreateADiskEncryptionSet
+    {
+        // Create a disk encryption set.
+        [NUnit.Framework.Test]
+        [NUnit.Framework.Ignore("Only verifying that the sample builds")]
```

Comment:
Do we need to declare the function as "[NUnit.Framework.Test]" for sample code?

Reply:
No, we do not have to.
But the Test attribute here is declaring that this method is a test and is a terminal method in case that when any one is writing something in this project and reference this method when he did not know what the purpose is for this method.
This is also an existing pattern for our hand written samples: https://github.com/Azure/azure-sdk-for-net/blob/ca3c6464db2b44a32e98ce8a7304129007f50a72/sdk/compute/Azure.ResourceManager.Compute/tests/Samples/Sample1_ManagingDisks.cs#L17